components/integration/ballerina-vs-apollo-for-graphql/code/apollo-graphql-bbe/connect-with-anything.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Connect with anything'
-description: 'Access thousands of connectors for HTTP APIs (OpenAPI), event APIs (AsyncAPI), GraphQL services, legacy systems, and data stores, allowing seamless data transfer to and from any system, anywhere.'
+description: 'Access thousands of connectors for HTTP APIs (OpenAPI), event APIs (AsyncAPI), GraphQL services, legacy systems, and data stores, allowing seamless data transfer to and from any system.'
```

Comment:
IMO, anywhere is needed to emphasize the difference. @ThisaruGuruge WDYT?

Reply:
Exactly! We should keep it IMO too.

Reply:
Reverted to the original.